README.md

```diff
@@ -390,7 +390,16 @@ Building the program with BLAS support may lead to some performance improvements
 
   Check [BLIS.md](docs/BLIS.md) for more information.
 
+- #### SYCL (using Intel DPCPP)
+  SYCL is a higher-level programming model to improve programming productivity on various hardware accelerators.
+
+  llama.cpp based on SYCL is used to **support Intel GPU** (Data Center Max series, Flex series, Arc series, Built-in GPU and iGPU).
+
+  For detailed info, please refer to [llama.cpp for SYCL](README_sycl.md).
```

Comment:
I believe this was changed on `master` to use a dash:

```suggestion
  For detailed info, please refer to [llama.cpp for SYCL](README-sycl.md).
```

Reply:
nice catch. thanks. it should be resolved in https://github.com/ggerganov/llama.cpp/pull/5228/commits/ddfbc23ca96f4b1afa66ce7df7caf54e1a10c745